modules/portunus.nix

```diff
@@ -226,5 +226,12 @@ in
       userFilter = replaceStr: "(|(uid=${replaceStr})(mail=${replaceStr}))";
       userBaseDN = "ou=users";
     };
+
+    systemd.services = lib.mkIf cfg.oauth2-proxy.configure {
```

Comment:
Unrelated? Otherwise looks fine

Reply:
Just sneaked in because I had it laying around locally.